aladdin-container.sh

```diff
@@ -17,17 +17,21 @@ export SCRIPT_DIR
 export ALADDIN_CONFIG_DIR
 export PY_MAIN
 
+source "$SCRIPT_DIR/shared.sh" # to load _extract_cluster_config_value
+
 # Test user's aws configuration
-function test_aws_config() {
+function _test_aws_config() {
+    # Test aws configuration for a given profile: $1
     echo "Testing aws configuration..."
+    local profile=$1
     # See if we the current aws profile configured
-    if ! aws configure list --profile "$AWS_PROFILE" &>/dev/null; then
-        echo "Could not find aws profile: $AWS_PROFILE; please check your ~/.aws/config and ~/.aws/credentials files"
+    if ! aws configure list --profile "$profile" >/dev/null; then
+        echo "Could not find aws profile: $profile; please check your ~/.aws/config and ~/.aws/credentials files"
         exit 1
     fi
     # Do a test aws cli call for the current aws profile
-    if ! aws s3 ls --profile "$AWS_PROFILE" &>/dev/null; then
-        echo "Your aws $AWS_PROFILE credentials or config may be malformed; please check your ~/.aws/config and ~/.aws/credentials files"
+    if ! aws sts get-caller-identity --profile "$profile" >/dev/null; then
```

Comment:
On further thought `aws sts get-caller-identity` is even better because it doesn't rely on any permissions, i.e. a user with just access keys and no perms will return something successful with this